### ISSUE TITLE
BUG: style parser doesn't handle \input properly

### ISSUE THREAD

**Issue by JonathanLevine657** [texstudio-org/texstudio#1551]
.cwl autogeneration fails for the `tkz-euclide` package, according to #752.

I investigated out of curiosity and the sty file for `tkz-euclide` defines no commands itself but instead includes two other files:
```tex
\input{tkz-tools-intersections}
\input{tkz-tools-angles}
```

I tried embedding these files directly into the sty and the autogenerator did quite a good job (though see related #1550). 

The cwl autogenerator should handle thse `\input` commands when parsing, it looks like an easy enhancement which will potentially apply to 3rd party packages.

**Comment by JonathanLevine657** [comment]
Also need to trigger cwl regeneration after users upgrade.

**Comment by sunderme** [comment]
> Also need to trigger cwl regeneration after users upgrade.

I don't think so
First there is no good way of detecting changes.
Second tex changes slowly. And up to now, there have not been complaints about this.

**Comment by JonathanLevine657** [comment]
> Second tex changes slowly. 
If a user uses a an affected package they already have a generated file, which will not be regenerated with the fixed parser. this fix cwl do nothing for them. What's does the velocity of tex have to do with it?

> And up to now, there have not been complaints about this.

I'm complaining. And the user who opened the bug complained. And neither of us will see the fix without regenerating the file. Other users are affected but not aware that the problem is actually a bug and not a missing CWL. you're asking too much.

> there is no good way of detecting changes.

clear the autogenerated directory the first time a newer version of the style parser code is run.

**Comment by sunderme** [comment]
I doubt that the issue has affected many users. txs code failed to recognized included files with a dash in the name.
The final goal is anyway, having edited cwl files. The autogeneration is just a last resort. You can provoke a regeneration by removing the cwl files in .config/.../autogenerated

**Comment by JonathanLevine657** [comment]
yes, I saw the commit and realized that was the problem and I've already regenerated the file as you suggest. But non-technical users who have seen orange keywords on their screen for months/years have no way of knowing that:
1. it's a bug
2. it's been fixed
3. they have to manually erase some file 3 levels deep in the config directory to benefit from the fix.

> I doubt that the issue has affected many users.

It's certainly affected *some* users, right?
